packages/server-wallet/src/errors.ts

```diff
@@ -0,0 +1,19 @@
+const CHANNEL_EXISTS = new Error(
+  'Attempting to open a channel that already exists -- use a different nonce'
+);
+const CHANNEL_MISSING = new Error('Channel does not exist');
+const STATE_NOT_SIGNED = new Error('State not signed by mover');
+const INVALID_TRANSITION = new Error('Invalid transition');
+const INVALID_STATE_UNKNOWN_REASON = new Error('State is not valid, but th reason is not known');
+const NOT_OUR_TURN = new Error('Not our turn to create a state');
+const VALUE_LOST = new Error('Value not preserved');
```

Comment:
Maybe exceptions should be `CapitalCamelCase`? Also shouldn't these values include some kind of parameters e.g., `channelId` or "error codes" associated with them (i.e., based on client-api-schema)

Reply:
https://github.com/statechannels/monorepo/issues/2325